scout/server/extensions/clinvar_extension.py

```diff
@@ -15,9 +15,9 @@ class ClinVarApi:
     the ClinVar submission schema (https://www.ncbi.nlm.nih.gov/clinvar/docs/api_http/) and if valid, can be directly submitted to ClinVar.
     """
 
-    def __init__(self):
+    def init_app(self, app):
```

Comment:
Has to be this way in order to be able to access the config file

Reply:
All good!